tempto-core/src/main/java/io/prestosql/tempto/context/TestContext.java

```diff
@@ -75,6 +75,8 @@ default TestContext createChildContext(State... states)
      */
```

Comment:
wow. This is quite sophisticated commit for what it does. I would expect simple `if` somewhere to NOOP on empty requirements.

Why do ne need additional method like ` void injectMembers(Object instance);`

Reply:
> Why do ne need additional method like void injectMembers(Object instance);

To not to rely on GuiceTestContext (TestContext implementation). So I could use `ForwardingTestContext`